AWS daily APIPermissionUpdate reconnect fails

**Summary of the defect**
The test failed due to PLATFORM_TRANSACTION_NOT_CREATED.


**How to reproduce (if possible)**

Please refer to the slack URL: https://hedera-hashgraph.slack.com/archives/CKWHL8R9A/p1606723088018100

and the detailed log file: https://s3.console.aws.amazon.com/s3/object/hedera-service-regression-jrs?region=us-east-2&prefix=4N_1C/Reconnect/20201130-0717-AWS-Daily-Services-Comp-Reconnect-4N-1C/APIPermissionUpdate-Reconnect-1-12m.json/node0000-TestClient/hapi-client-combined.log


**Service logs (if applicable)**
```
...

2020-11-30 07:46:20.267 INFO   128  HapiQueryOp - 'updateApiPermissionsDuringReconnect' - Paying for HapiGetFileContents{sigs=1, payer=strong-control, node=0.0.6, file=API_PERMISSIONS, fileId=fileNum: 122
} with body=transactionID { transactionValidStart { seconds: 1606722320 nanos: 380 } accountID { accountNum: 50 } } nodeAccountID { accountNum: 6 } transactionFee: 84163 transactionValidDuration { seconds: 120 } memo: "\303\203\302\256\303\202\302\267\303\203\302\271tF8\303\202\302\256J\303\203\302\213\303\203\302\220\303\203\302\216" cryptoTransfer { transfers { accountAmounts { accountID { accountNum: 50 } amount: -8 } accountAmounts { accountID { accountNum: 6 } amount: 8 } } }; sigs=sigPair { pubKeyPrefix: "\n" ed25519: "\372l]\231|*\257\035e\212\241?\200\265Q~u\000I\005J\244\202cf\244\242_\024-\344\367\351\365xs9\273OB=\226?#\220\371\326\214\243\tdW\274\273^\272\036\225\312\221C\302\t\b" }
2020-11-30 07:46:20.383 INFO   155  HapiGetFileContents - API_PERMISSIONS contained 0 bytes
2020-11-30 07:46:20.383 INFO   164  HapiGetFileContents - As a config list, contents are:
2020-11-30 07:46:20.385 WARN   199  HapiSpecOperation - 'updateApiPermissionsDuringReconnect' - HapiGetFileContents{sigs=1, payer=strong-control, node=0.0.6, file=API_PERMISSIONS, fileId=fileNum: 122
} failed!
java.lang.AssertionError: Bad answerOnlyPrecheck! expected:<OK> but was:<PLATFORM_TRANSACTION_NOT_CREATED>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at org.junit.Assert.failNotEquals(Assert.java:834) ~[SuiteRunner.jar:?]
	at org.junit.Assert.assertEquals(Assert.java:118) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.queries.HapiQueryOp.submitOp(HapiQueryOp.java:144) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.exec(HapiApiSpec.java:222) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.run(HapiApiSpec.java:176) ~[SuiteRunner.jar:?]
	at java.util.AbstractList$RandomAccessSpliterator.forEachRemaining(AbstractList.java:720) ~[?:?]
	at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:658) ~[?:?]
	at com.hedera.services.bdd.suites.HapiApiSuite.runSync(HapiApiSuite.java:249) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.suites.HapiApiSuite.runSuite(HapiApiSuite.java:140) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.suites.HapiApiSuite.runSuiteSync(HapiApiSuite.java:133) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.suites.SuiteRunner.lambda$runSuitesSync$15(SuiteRunner.java:490) ~[SuiteRunner.jar:?]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:176) ~[?:?]
	at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948) ~[?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[?:?]
	at com.hedera.services.bdd.suites.SuiteRunner.runSuitesSync(SuiteRunner.java:491) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.suites.SuiteRunner.lambda$runTargetCategories$10(SuiteRunner.java:465) ~[SuiteRunner.jar:?]
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195) [?:?]
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1654) [?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) [?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) [?:?]
	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) [?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) [?:?]
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) [?:?]
	at com.hedera.services.bdd.suites.SuiteRunner.runTargetCategories(SuiteRunner.java:465) [SuiteRunner.jar:?]
	at com.hedera.services.bdd.suites.SuiteRunner.runCategories(SuiteRunner.java:434) [SuiteRunner.jar:?]
	at com.hedera.services.bdd.suites.SuiteRunner.main(SuiteRunner.java:358) [SuiteRunner.jar:?]
2020-11-30 07:46:20.390 INFO   244  HapiApiSpec - 'updateApiPermissionsDuringReconnect' - final status: FAILED!
2020-11-30 07:46:20.391 INFO   168  UpdateApiPermissionsDuringReconnect - -------------- RESULTS OF UpdateApiPermissionsDuringReconnect SUITE --------------
2020-11-30 07:46:20.391 INFO   170  UpdateApiPermissionsDuringReconnect - Spec{name=updateApiPermissionsDuringReconnect, status=FAILED}
2020-11-30 07:46:20.395 INFO   439  SuiteRunner - ============== sync run results ==============
2020-11-30 07:46:20.395 INFO   442  SuiteRunner - UpdateApiPermi...uringReconnect :: 0/1 suites ran OK
2020-11-30 07:46:20.395 INFO   448  SuiteRunner -   --> Problems in suite 'UpdateApiPermissionsDuringReconnect' :: Spec{name=updateApiPermissionsDuringReconnect, status=FAILED}


```
**Environment:**
- OS: [e.g. Ubuntu 18.04]
- Java: [e.g. OpenJDK 11.0.4]
- Hedera Services Version: [e.g. 0.0.5]
- HAPI Version: [e.g. 0.0.5]

**Additional Context**
Add any other context about the problem here. Attach any logs here, if applicable.

--- a/test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/UpdateApiPermissionsDuringReconnect.java
+++ b/test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/UpdateApiPermissionsDuringReconnect.java
@@ -29,7 +29,9 @@ import org.apache.logging.log4j.Logger;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
+import java.time.Duration;
 
+import static com.hedera.services.bdd.spec.utilops.UtilVerbs.sleepFor;
 import static com.hedera.services.bdd.spec.HapiApiSpec.defaultHapiSpec;
 import static com.hedera.services.bdd.spec.queries.QueryVerbs.getAccountBalance;
 import static com.hedera.services.bdd.spec.queries.QueryVerbs.getFileContents;
@@ -54,6 +56,7 @@ public class UpdateApiPermissionsDuringReconnect extends HapiApiSuite {
 		final String fileInfoRegistry = "apiPermissionsReconnect";
 		return defaultHapiSpec("updateApiPermissionsDuringReconnect")
 				.given(
+						sleepFor(Duration.ofSeconds(25).toMillis()),
 						getAccountBalance(GENESIS).setNode("0.0.6").unavailableNode()
 				)
 				.when(

